### ISSUE TITLE
Compiling error: cannot create directory

### ISSUE THREAD

**Issue by ngdangtu-vn** [msharov/snownews#49]
I've tried to compile and package with these commands:

```
sudo auto-apt run ./configure
sudo checkinstall
```

And the result is it failed:
```
========================= Installation results ===========================
Installing /usr/local/share/man/man1/snownews.1 ...
install: cannot create directory '/usr/local/share/man': No such file or directory
make: *** [man/Module.mk:18: /usr/local/share/man/man1/snownews.1] Error 1

****  Installation failed. Aborting package creation.

Cleaning up...OK

Bye.
```

What I expect is it should create a `deb` file and install it to my OS.

Here is my PC info:
- `uname -a`
   ```
   Linux *** 5.4.51-v7l+ #1333 SMP Mon Aug 10 16:51:40 BST 2020 armv7l GNU/Linux
   ```
- ARMHF architecture
- Raspberry Pi 4 (8GB)
- snownews : clone directly the latest version from master branch (date: 9 Sep, 2020)

**Comment by ngdangtu-vn** [comment]
Most of the missing dirs are about language pack. (`man` and `locale`)

**Comment by msharov** [comment]
Closed by commit 268ef84660c03df29e8f155c3570092add1596cd
